sips/sip-019/sip-019-token-metadata-update-notifications.md

```diff
@@ -212,6 +212,26 @@ Notifications that do not meet these requirements must be ignored.
 Given these constraints the notifications this SIP proposes should be taken as _hints_ to metadata
 indexers. Metadata indexers are not obliged to follow them.
 
+# Related work
+
+An alternative considered for token metadata update notifications is for them to be transmitted via
+an off-chain notification service such as an official mailing list or a forum post. While this
+channel would have had the advantage of being easier to update and faster to propagate, it has a few
+key disadvantages that make it inadequate for this SIP's intended use:
+
+1. It introduces a third party dependency
+    * An off-chain notification service would most likely be maintained by centralized entities
+      unrelated to the Stacks ecosystem which at any time may modify the channel, its reach, or its
+      rules.
+1. It does not leave a permanent record in the blockchain
```

Comment:
I'm not sure why this point is important?  If the SIP is treating metadata updates as hints -- i.e. indexers decide how strictly they follow them -- then does permanence matter?  Do Clarity contracts need access to updates?  Do indexers need access to the history of updates?